dm/worker/config.go

```diff
@@ -191,6 +199,9 @@ func (c *Config) Parse(arguments []string) error {
 	}
 	c.From.Password = pswd
 
+	// assign tracer id to source id
+	c.Tracer.Source = c.SourceID
```

Comment:
can use set `Source` directly? if not, change 
```
Source     string `toml:"source" json:"source"`    
```
to
```
Source     string `json:"source"`    
```
?

Reply:
It is not so important, if user set a `Source` it will always be overwrite, and it is convenient to leave this if we want to encode toml of tracer config

Reply:
fine